### PR TITLE
Make components and block optional

### PR DESCRIPTION
**Pull request by DjidjaleskaSandra**
# Description

This ensures that the registration process can continue without being interrupted by potential errors related to missing or invalid component/blocks paths.

# Productive ticket
https://app.productive.io/1-infinum/tasks/7820880

### CODE REVIEW

**Comment by iruzevic** [comment]
I see we have other stuff here:

```
registerBlocks(
	globalSettings,
	Wrapper,
	WrapperManifest,
	require.context('./../../components', true, /manifest.json$/),
	require.context('./../../custom', true, /manifest.json$/),
	require.context('./../../custom', true, /-block.js$/),
	require.context('./../../custom', true, /-hooks.js$/),
	require.context('./../../custom', true, /-transforms.js$/),
	require.context('./../../custom', true, /-deprecations.js$/),
	require.context('./../../custom', true, /-overrides.js$/),
);

registerVariations(
	globalSettings,
	require.context('./../../variations', true, /manifest.json$/),
	require.context('./../../custom', true, /manifest.json$/),
	require.context('./../../variations', true, /overrides.json$/),
);
```


Pleae check all other stuff here like variations and other .js files also will it work

**Review by goranalkovic-infinum — DISMISSED** [review]

File: blocks/init/src/Blocks/assets/scripts/application-blocks-editor.js [hunk 1]
```diff
@@ -23,24 +23,44 @@ import { Wrapper } from '../../wrapper/wrapper';
 import WrapperManifest from '../../wrapper/manifest.json';
 import globalSettings from '../../manifest.json';
 
+const safelyRequireContext = (basePath, searchSubdirectories, regularExpression) => {
```
Comment (iruzevic) [inline blocks/init/src/Blocks/assets/scripts/application-blocks-editor.js:26]: The idea was not to change anything in the existing project's implementation. Can this be done without changing the existing files?

**Review by iruzevic — CHANGES_REQUESTED** [review]

File: blocks/init/src/Blocks/assets/scripts/application-blocks-editor.js (hunk 1 above)
Comment (DjidjaleskaSandra) [reply blocks/init/src/Blocks/assets/scripts/application-blocks-editor.js:26]: Could you please clarify what you mean by "not changing anything in the existing project's implementation"? I have no any other idea  :)

Comment (iruzevic) [reply blocks/init/src/Blocks/assets/scripts/application-blocks-editor.js:26]: We have this file:
https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/assets/scripts/application-blocks-editor.js

I want it to remain exactly as is. So all changes you can make is inside the `registerBlocks` and `registerVariations` methods

**Pull request closed without merging**